### PR TITLE
Sample bike data displayed on map

### PR DESCRIPTION
**Pull request by jonramm**
- Add sample bike data
- Encapsulate Map as separate component
- Create BikeMarker component
- Pull in sample bike data and display as markers on map

### CODE REVIEW

File: client/components/BikeMarker.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    status: string,
+    lock_combo: string,
+    location: LocationProps,
+    tags: string[]
+}
+
+const BikeMarker = (props: BikeProp) => {
+    return (
+        <Marker 
+        key="1"
```
Comment (lyanita) [inline client/components/BikeMarker.tsx]: key={props.bike.bike_id}

If not, you get:
Warning: Each child in a list should have a unique "key" prop.

File: client/components/Map.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import MapView from 'react-native-maps';
+import { 
+    StyleSheet, 
+} from 'react-native';
+import MyMarker from "./MyMarker";
+import BikeMarker from "./BikeMarker";
+import { bikes } from '../data/testData';
+
+type LocationProps = {
```
Comment (lyanita) [inline client/components/Map.tsx]: Just a suggestion: The LocationProps type object is found in both Map.tsx and BikeMarker.tsx. If they are going to be repeatedly used throughout multiple components, maybe create a types/interface file/folder to store them once, then import as desired, to keep things DRY.

**Review by lyanita — DISMISSED** [review]
Feel free to merge, just some minor changes/suggestions made.

File: client/components/BikeMarker.tsx (hunk 1 above)
Comment (jonramm) [reply client/components/BikeMarker.tsx]: Fixed by adding 'key' prop to map function in Map.tsx.

File: client/components/Map.tsx (hunk 2 above)
Comment (jonramm) [reply client/components/Map.tsx]: Made a types.d.ts file to hold all app types to import .

**Comment by jonramm** [comment]
New PR contains these changes

**Pull request closed without merging**